lib/rules/define-macros-order.js

```diff
@@ -172,6 +187,58 @@ function create(context) {
     })
   }
 
+  /**
+   * @param {ASTNode} node
+   * @param {ASTNode} lastNode
+   */
+  function reportExposeNotOnBottom(node, lastNode) {
+    context.report({
+      node,
+      loc: node.loc,
+      messageId: 'defineExposeNotTheLast',
+      fix(fixer) {
+        return moveNodeToLast(fixer, node, lastNode)
```

Comment:
I think that probably shouldn't support auto-fix. It may break user's code.

```
<script setup>
let a = 1
defineExpose({a})
a = 2
</script>
```

What do you think about supporting suggestions instead?

Reply:
Sorry for the late response. It seems reasonable to me not to support auto-fix, thanks for pointing out!
I'll add to `suggestion` and refine the test cases.